integrated_channels/catalog_service_utils.py

```diff
@@ -0,0 +1,25 @@
+"""
+A utility collection for calls from integrated_channels to Catalog service
+"""
+from enterprise.api_client.discovery import get_course_catalog_api_service_client
+
+
+def get_course_id_for_enrollment(enterprise_enrollment):
+    """
+    Fetch course_id for a given enterprise enrollment
```

Comment:
For both functions please add a docstring about what happens if one is not found (since this is a remote client): specific exception, returns none?